Lib/email/contentmanager.py

```diff
@@ -146,7 +146,7 @@ def embedded_body(lines): return linesep.join(lines) + linesep
     def normal_body(lines): return b'\n'.join(lines) + b'\n'
     if cte==None:
         # Use heuristics to decide on the "best" encoding.
-        if max(len(x) for x in lines) <= policy.max_line_length:
+        if max((len(x) for x in lines), default = 0) <= policy.max_line_length:
```

Comment:
```suggestion
        if max((len(x) for x in lines), default=0) <= policy.max_line_length:
```
NIT: PEP 8 advice not using space for keyword arguments: https://www.python.org/dev/peps/pep-0008/#other-recommendations